### DISCUSSION TITLE
Does the plugin work with everything service?

### DISCUSSION THREAD

**Discussion by LucasM314** [lin-ycv/EverythingPowerToys#143] [Q&A] [answered]
Hey!
Right now, the plugin works perfectly fine if Everything is running in the background.
However, I tried stopping the Everything background process while letting the Everything service running, and the plugin didn't work anymore.
Is it expected behaviour?

**Comment by lin-ycv** [comment, marked as answer]
Everything service is for indexing only. 
https://www.voidtools.com/support/everything/everything_service/
So you have to have Everything, the main program, running to be able to search with EPT.

**Reply by LucasM314** [reply]
Ah, I see, thanks!
